daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_dev/daml_lf_1.proto

```diff
@@ -274,6 +274,10 @@ enum PrimType {
   // Builtin type 'Any'
   // *Available in versions >= 1.dev*
   ANY = 18;
```

Comment:
Did you forget to add the equality primitive here?

Reply:
Indeed.

Reply:
Fixing that now.